news/2018-02-25-project-loved-week-of-february-25th.md

```diff
@@ -0,0 +1,287 @@
+
+[![](/wiki/shared/news/2018-02-25-project-loved-week-of-february-25th/mania/the-lost-dedicated.jpg)](https://osu.ppy.sh/community/forums/topics/708939)
+
+*written by [Shoegazer](https://osu.ppy.sh/users/2520707)*
+
+The next chart on our list is one of the most recognisable charts in competitive circles in osu!mania: [Zyph](https://osu.ppy.sh/users/1600432)'s [The Lost Dedicated](https://osu.ppy.sh/beatmapsets/224173). As a very clear allusion to the "life" difficulty in BMS, this chart contains overwhelmingly dense chordjacks. However, the chart seems to convey the track quite well despite its level of density. Its density is substantial, yet meticulously controlled. It also captures certain sections of the track very nicely with completely different but very complementary patterning, which is always impressive when you're dealing with densities this high.
+
+The chart's overwhelming popularity, current reception and level of nuance are all reasons that we believe that this chart should be in the Loved section.
+
+[![](/wiki/shared/news/2018-02-25-project-loved-week-of-february-25th/mania/the-glory-days.jpg)](https://osu.ppy.sh/community/forums/topics/708940)
+
+*written by [Kamikaze](https://osu.ppy.sh/users/2124783)*
+
+Up next we have [The Glory Days](https://osu.ppy.sh/beatmapsets/373357), one of the most popular unranked charts in 2016, a full spread mapset by a well known Korean charter [CloudHolic](https://osu.ppy.sh/users/3924047), known mostly for his mixed style and fun oriented patterning.
+
+The Glory Days is a map that's very accessible, with difficulties for all kinds of players, from a fairly standard normal which you can train 7k on if you're new to it to very dense and tricky Glory difficulty that challenges you in multitude of ways.
+
+The highest difficulty is a mix of very dense chordstreaming with small bursts in the choruses, that are also hardened by the usage of sporadic long LNs and calmer parts which are more LN oriented with some tricky to nail transitions making it an interesting map all around and a good choice for the loved category.
+
+[![](/wiki/shared/news/2018-02-25-project-loved-week-of-february-25th/mania/river-flows-in-you.jpg)](https://osu.ppy.sh/community/forums/topics/708941)
+
+*written by [Kamikaze](https://osu.ppy.sh/users/2124783)*
+
+This set is something special. Brought by our own Elite Mapper [LordRaika](https://osu.ppy.sh/users/3265023), it is a mapset that really nicely conveys what kind of a charter LordRaika is.
+
+Not only is this mapset keysounded, the song itself is LordRaika's mix of it, and it also has a wonderful storyboard by the elite storyboarder [Exile-](https://osu.ppy.sh/users/2559349).
+
+It doesn't end there! Because the mapset also offers an immense set of 30 difficulties throughout all 6 rankable keymodes, from 4k to 9k, offering a wide range of patterns and densities to train your favourite keymode or perhaps, learn a new one!
```

Comment:
I assume that this was a copy-paste thing from the people who actually wrote it, but the casing of the K's for the key counts are inconsistent